### DISCUSSION TITLE
maa什么时候更新一下两班倒的默认基建换班方案

### DISCUSSION THREAD

**Discussion by Flame38** [MaaAssistantArknights/MaaAssistantArknights#7694] [General]
如题，自己配的基建排班表不知道为什么老是有问题，而且里面的243三班倒也已经是重岳版本的，期待更新：）

**Comment by Constrat** [comment]
You don't know why there are issues and you are expecting us to somehow update it and fix it?
Custom infrastructure schemes are something the user needs to do on their own.
(Especially considering the amount of new operators and new base possibilities)
